examples/inferencing/running-on-gke.md

```diff
@@ -121,13 +134,15 @@ A GCS path to upload the model after it is converted for FasterTransformer
   </tr>
 </table>
 
-
+Note: The sample configuration still requires the user to specify a `MODEL_SOURCE_PATH` variable.
```

Comment:
Do we support direct download from HuggingFace? If so, the default config should download flan-t5 (preferrably XXL). We can then remove this note.

Reply:
We do. I'll set it to flan-t5-base so that it's still functional until the inferencing issues are resolved.

Reply:
Great! We can remove this Note now.

Reply:
Ah yep, fixed.